flocker/provision/_install.py

```diff
@@ -376,7 +388,7 @@ def task_upgrade_kernel(distribution):
     """
     Upgrade kernel.
     """
-    if distribution == 'centos-7':
+    if is_centos(distribution):
```

Comment:
It's `admin.DISTRIBUTION_NAME_MAP` and `flocker.provision` can't depend on `admin`.  Also probably not really worthwhile considering we should be replacing `flocker.provision` with something not hand-rolled as soon as we can.